coderd/userauth.go

```diff
@@ -921,7 +921,8 @@ func (api *API) oauthLogin(r *http.Request, params oauthLoginParams) (*http.Cook
 					Username:       params.Username,
 					OrganizationID: organizationID,
 				},
-				LoginType: params.LoginType,
+				CreateOrganization: len(organizations) == 0,
```

Comment:
I don't think this should ever be possible to evaluate to true so it may be good to comment why we are doing this